### ISSUE TITLE
Ignore agconnect-services.json file in non-HVR builds

### ISSUE THREAD

**Issue by svillar** [Igalia/wolvic#283]
For non-HVR builds we should completely ignore that file. Currently build fails if some other flavour (i.e. Meta) is built while the file is in place.

I think this is great for someone from the community to fix.

**Comment by svillar** [comment]
BTW the issue is actually while running the application not at build time. This is an example of the error I get
`Caused by org.gradle.api.GradleException: ERROR: Failed to verify AGConnect-Config '/client/package_name', expected: 'com.igalia.wolvic.dev', but was 'com.igalia.wolvic'`

I was indeed building a debug build of Wolvic without removing the agconnect-services.json file from a previous release build for HVR.

**Comment by DevPika** [comment]
I am attempting to fix this issue. I believe the root cause of this is [L#10 in `/app/build.gradle`](https://github.com/Igalia/wolvic/blob/359c9df3838e11458c0e0fa4e40fb420acaebfdd/app/build.gradle#L10). 
```
apply plugin: 'com.huawei.agconnect'
```
As this is applied to the whole project, even non-HVR builds try to use the plugin. I tried moving `apply plugin: 'com.huawei.agconnect'` to `hvr` `productFlavour` but that still seems to add the plugin to the whole project. I followed [another approach](https://stackoverflow.com/a/43336869), using the following conditional inside `build.gradle` works:
```
if (getGradle().getStartParameter().getTaskRequests()
        .toString().contains("Hvr")){
    apply plugin: 'com.huawei.agconnect'
}
```
I have tested this with a build for Meta Quest 2 - `agconnect-services.json` is ignored even if it exists. I don't have access to the HVR SDK to test this though. I will open an MR for this, please test with HVR builds.

**Comment by svillar** [comment]
Already fixed in #389
